src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,725 @@
+'aminoacylation reaction'.
+Therefore, other vocabularies like the `Gene Ontology (GO) <https://www.ebi.ac.uk/QuickGO/>`_ or the
+`Molecular Process Ontology (MOP) <https://www.ebi.ac.uk/ols/ontologies/mop>`_ were used to find corresponding
+interaction terms. These terms were then annotated with the name, namespace and identifier.
+For negative protein modifications in which a group is split from the protein like `decarboxylation reaction
+<https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_1140>`_,
+the positive term `protein carboxylation <https://www.ebi.ac.uk/QuickGO/term/GO:0018214>`_ is taken and a interaction
+describing the decrease of the target is taken.
+In the case of `gtpase reaction <https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_0883>`_ and
+`atpase reaction <https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_0882>`_,
+the notion of the source protein taking on the ability to catalyze a GTP or ATP hydrolysis had to  be mentioned.
+Therefore, :func:`pybel.dsl.activity` was added as the subject_modifier of the source protein.
+A very special case was that of the `dna strand elongation
+<https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_0701>`_ .
+
+Here, the target was a gene and to capture the notion of the DNA strand elogation process, the corresponding GO term
+was added as a :class:`pybel.dsl.GeneModification`. In the case of DNA or RNA cleavage, the target was set as the entity
+of :class:`pybel.dsl.Gene` or :class:`pybel.dsl.Rna`.
+For the relation `isomerase reaction
+<https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_1250>`_
+there was no corresponding term in BEL denoting this process. Therefore, the molecular process `isomerization
+<https://www.ebi.ac.uk/ols/ontologies/mop/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMOP_0000789>`_ from the
+`MOP <https://www.ebi.ac.uk/ols/ontologies/mop>`_ was used and annotated.
+
+As IntAct and BioGRID are both interaction databases, the general code from biogrid.py could be taken as an initial
+approach. Due to the higher granularity of IntAct concerning the interaction types, many modifications and special
+cases as mentioned above had to be further investigated and were applied case-sensitive.
+
+Moreover, a very interesting type of information in IntAct is the negative interaction data which means that a target
+would not be activated by the source. This type of relations could also be mapped to negative BEL. In machine learning
+tasks like link prediction in graphs these negative edges could be used as negative samples to enhance the prediction
+quality of the model.
+
+Complexes are also used in IntAct and documented with an internal IntAct ID. These complexes were not taken into account
```

Comment:
re-phrase to talk about this being a future improvement

keep in mind the fact that you only care about the uniprot identified entities doesn't mean someone else who might want IntAct as BEL wouldn't

Reply:
Wouldn't care about the uniprot identifiers you mean?

Reply:
the specific use case for your master's thesis isn't the point of the documentation here. You have the chance to outline where the code is insufficient and will be improved at a later time; so you just have to reword. Something like this might be better:

"IntAct also gives internal accession numbers to some complexes, but IntAct does not yet map to preferred resources like ComplexPortal, so the code does not output them yet." Then reference ongoing discussion on twitter 🤣 :

<blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://twitter.com/bmeldal?ref_src=twsrc%5Etfw">@bmeldal</a> <a href="https://twitter.com/complexportal?ref_src=twsrc%5Etfw">@complexportal</a> <a href="https://twitter.com/reactome?ref_src=twsrc%5Etfw">@reactome</a> is there a place to download identifier mappings between reactome and complexportal identifiers? I can’t find it on Reactome’s downloads section but it’s definitely there (see: <a href="https://t.co/sOHNkdOKJB">https://t.co/sOHNkdOKJB</a>)</p>&mdash; Charles Tapley Hoyt (@cthoyt) <a href="https://twitter.com/cthoyt/status/1252345260740456453?ref_src=twsrc%5Etfw">April 20, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>